diff --git a/browser/branding/unofficial/configure.sh b/browser/branding/unofficial/configure.sh
index f42e11ddad..3cbfeb6ba9 100644
--- a/browser/branding/unofficial/configure.sh
+++ b/browser/branding/unofficial/configure.sh
@@ -2,5 +2,5 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
-MOZ_APP_DISPLAYNAME=Nightly
-MOZ_MACBUNDLE_ID=nightlyunofficial
+MOZ_APP_DISPLAYNAME="CanadaFox"
+MOZ_MACBUNDLE_ID=canadafox
diff --git a/browser/branding/unofficial/locales/en-US/brand.ftl b/browser/branding/unofficial/locales/en-US/brand.ftl
index 5eaea9a7c4..16f95ab628 100644
--- a/browser/branding/unofficial/locales/en-US/brand.ftl
+++ b/browser/branding/unofficial/locales/en-US/brand.ftl
@@ -16,12 +16,12 @@
 ## For further details, consult:
 ## https://mozilla-l10n.github.io/styleguides/mozilla_general/#brands-copyright-and-trademark
 
--brand-shorter-name = Nightly
--brand-short-name = Nightly
--brand-shortcut-name = Nightly
--brand-full-name = Nightly
+-brand-shorter-name = CanadaFox
+-brand-short-name = CanadaFox
+-brand-shortcut-name = CanadaFox
+-brand-full-name = CanadaFox
 # This brand name can be used in messages where the product name needs to
 # remain unchanged across different versions (Nightly, Beta, etc.).
--brand-product-name = Firefox
--vendor-short-name = Mozilla
+-brand-product-name = CanadaFox
+-vendor-short-name = CanadaFox
 trademarkInfo = { " " }
diff --git a/browser/branding/unofficial/locales/en-US/brand.properties b/browser/branding/unofficial/locales/en-US/brand.properties
index 346c3bdb50..494bef8000 100644
--- a/browser/branding/unofficial/locales/en-US/brand.properties
+++ b/browser/branding/unofficial/locales/en-US/brand.properties
@@ -2,6 +2,6 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
-brandShorterName=Nightly
-brandShortName=Nightly
-brandFullName=Nightly
+brandShorterName=CanadaFox
+brandShortName=CanadaFox
+brandFullName=CanadaFox
diff --git a/browser/config/version.txt b/browser/config/version.txt
index c60c42f574..cb05d38943 100644
--- a/browser/config/version.txt
+++ b/browser/config/version.txt
@@ -1 +1 @@
-153.0.4
+0.0.3.7
diff --git a/browser/config/version_display.txt b/browser/config/version_display.txt
index c60c42f574..cb05d38943 100644
--- a/browser/config/version_display.txt
+++ b/browser/config/version_display.txt
@@ -1 +1 @@
-153.0.4
+0.0.3.7
