diff --git a/browser/base/content/aboutDialog.css b/browser/base/content/aboutDialog.css
index 017125bc25..ee9cd81778 100644
--- a/browser/base/content/aboutDialog.css
+++ b/browser/base/content/aboutDialog.css
@@ -87,6 +87,13 @@
   margin-block: 0;
 }
 
+#sblTagline {
+  font-style: italic;
+  opacity: 0.75;
+  margin-block: 4px 10px;
+  margin-inline-start: 0;
+}
+
 .text-blurb {
   margin-bottom: 10px;
   margin-inline-start: 0;
diff --git a/browser/base/content/aboutDialog.xhtml b/browser/base/content/aboutDialog.xhtml
index 067eaf8d4e..ecba0e653c 100644
--- a/browser/base/content/aboutDialog.xhtml
+++ b/browser/base/content/aboutDialog.xhtml
@@ -14,7 +14,7 @@
         data-l10n-id="aboutDialog-title"
 #endif
         role="dialog"
-        aria-describedby="version distribution distributionId communityDesc contributeDesc trademark"
+        aria-describedby="version distribution distributionId sblTagline communityDesc contributeDesc trademark"
         >
 #ifdef XP_MACOSX
 #include macWindow.inc.xhtml
@@ -55,6 +55,7 @@
 
         <label id="distribution" class="text-blurb"/>
         <label id="distributionId" class="text-blurb"/>
+        <description id="sblTagline" data-l10n-id="sbl-about-tagline"/>
 
         <vbox id="detailsBox">
           <hbox id="updateBox">
diff --git a/browser/locales/en-US/browser/aboutDialog.ftl b/browser/locales/en-US/browser/aboutDialog.ftl
index 496f125ebc..693986f81e 100644
--- a/browser/locales/en-US/browser/aboutDialog.ftl
+++ b/browser/locales/en-US/browser/aboutDialog.ftl
@@ -5,6 +5,8 @@
 aboutDialog-title =
     .title = About { -brand-full-name }
 
+sbl-about-tagline = Because the Internet won’t browse itself!
+
 releaseNotes-link = What’s new
 
 update-checkForUpdatesButton =
